README.md

```diff
@@ -1,79 +1,66 @@
 # Microsoft Teams JavaScript client SDK
 
-Welcome to the Teams client SDK monorepo! For breaking changes, please refer to our changelog in the monorepo root of the `2.0-preview` branch.
-
-This JavaScript library is part of the [Microsoft Teams developer platform](https://developer.microsoft.com/microsoft-teams/). See full [SDK reference documentation](https://docs.microsoft.com/en-us/javascript/api/overview/msteams-client).
-
-[![Build Status](https://travis-ci.org/OfficeDev/microsoft-teams-library-js.svg?branch=master)](https://travis-ci.org/OfficeDev/microsoft-teams-library-js)
-[![Coverage Status](https://coveralls.io/repos/github/OfficeDev/microsoft-teams-library-js/badge.svg?branch=master)](https://coveralls.io/github/OfficeDev/microsoft-teams-library-js?branch=master)
+Welcome to the Teams client SDK monorepo! For breaking changes, please refer to our changelog in the monorepo root of the `2.0-preview` branch. This repository contains the core teams-client-sdk as well as tools and applications for analyzing and testing.
 
 ## Getting Started
 
-1. Clone the repo
-2. Run `yarn install` from repo root
-3. Run `yarn build` from repo root
-4. To run Unit tests, run `yarn test`
+ramourt 10 minutes ago •
+edited
+Member
 
-  TIP: whenever building or testing the Teams client SDK, you can run `yarn build` or `yarn test` from the packages/teams-js directory.
+We should keep this with the 4 instructions and the tip. Above the list of apps and under the coverage thing
 
-## Installation
+Jessica-Koch 13 seconds ago Author Member
 
-To install the latest 2.0 preview version:
+oh i totally agree, this was a mistake on my part
 
-### npm
+    Reply…
 
-`npm install --save @microsoft/teams-js@next`
+Resolve conversation
 
-### yarn
+1. Clone the repo
+2. Run `yarn install` from repo root
+3. Run `yarn build` from repo root
+4. To run Unit tests, run `yarn test`
 
-`yarn add @microsoft/teams-js@next`
+TIP: whenever building or testing the Teams client SDK, you can run `yarn build` or `yarn test` from the `packages/teams-js` directory.
 
-### Production
+This JavaScript library is part of the [Microsoft Teams developer platform](https://developer.microsoft.com/microsoft-teams/). See full [SDK reference documentation](https://docs.microsoft.com/en-us/javascript/api/overview/msteams-client).
```

Comment:
Can you please update the link to this:

https://docs.microsoft.com/en-us/javascript/api/overview/msteams-client?view=msteams-client-js-beta

That will ensure the beta version of the SDK docs is shown.

Reply:
yes!  this was what i was talking about in the conversation we had offline, thank you will do!